instrumentation/jetty-8.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/jetty/JettyQueuedThreadPoolModule.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.jetty;
+
+import static java.util.Collections.singletonList;
+import static java.util.Collections.singletonMap;
+import static net.bytebuddy.matcher.ElementMatchers.any;
+import static net.bytebuddy.matcher.ElementMatchers.named;
+import static net.bytebuddy.matcher.ElementMatchers.takesArgument;
+import static net.bytebuddy.matcher.ElementMatchers.takesArguments;
+
+import com.google.auto.service.AutoService;
+import io.opentelemetry.javaagent.instrumentation.api.ContextStore;
+import io.opentelemetry.javaagent.instrumentation.api.InstrumentationContext;
+import io.opentelemetry.javaagent.instrumentation.api.Java8BytecodeBridge;
+import io.opentelemetry.javaagent.instrumentation.api.concurrent.ExecutorInstrumentationUtils;
+import io.opentelemetry.javaagent.instrumentation.api.concurrent.RunnableWrapper;
+import io.opentelemetry.javaagent.instrumentation.api.concurrent.State;
+import io.opentelemetry.javaagent.tooling.InstrumentationModule;
+import io.opentelemetry.javaagent.tooling.TypeInstrumentation;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import net.bytebuddy.asm.Advice;
+import net.bytebuddy.description.method.MethodDescription;
+import net.bytebuddy.description.type.TypeDescription;
+import net.bytebuddy.matcher.ElementMatcher;
+
+@AutoService(InstrumentationModule.class)
+public class JettyQueuedThreadPoolModule extends InstrumentationModule {
```

Comment:
Why not keep them separate?  Is it a problem to do that?  If we do combine them, we'll want to rename the existing module then probably from `JettyHandlerInstrumentationModule` to just `JettyInstrumentationModule` or something because it will no longer be handler specific.

I can make that change, just not clear on the advantage.  I'm also uncertain about bringing over the `contextStore()` override since it doesn't necessarily seem to apply to jetty handlers.

Reply:
the advantage of having a single `InstrumentationModule` is that we know either all or none of the instrumentation will be applied (not muzzled), which reduces the number of states we have to think about when developing or troubleshooting

the `contextStore()` is ok, at the `InstrumentationModule` level it's just a combination of all context stores needed for instrumentation in the module